system.prop

```diff
@@ -16,3 +16,22 @@ lockscreen.rot_override=true
 ### WIFI
 wifi.direct.interface=p2p0
 ro.boot.wificountrycode=00
+
+# Free up RAM by purging assets.
+persist.sys.purgeable_assets=1
+
+# Better RAM management
+ro.HOME_APP_ADJ=1
```

Comment:
Also this isn't related to "RAM management"
It's for pinning home app to memory on **older android versions** and as such it even does the opposite of what you wanted to do (if it still worked on Q anyways)

Reply:
It says # Better RAM management?

Reply:
But that's not what it actually does as I said.

Reply:
> But that's not what it actually does as I said.

I looked it up, apparently, it is supposed to add more files to your RAM.